source/MaterialXView/Main.cpp

```diff
@@ -265,6 +281,8 @@ int main(int argc, char* const argv[])
                                             screenColor);
         viewer->setMeshRotation(meshRotation);
         viewer->setMeshScale(meshScale);
+        viewer->setTurntableIncrement(360.0f / static_cast<float>(turntableSteps));
```

Comment:
In addition to describing the user input in terms of turntable steps, I was also imagining that the internal logic would operate in terms of integral current/total steps, eliminating the need to accumulate the rotation angle as a series of floating-point additions.  This would remove the potential for floating-point precision issues when the increment angle doesn't turn out to be exactly representable as a float.

Reply:
Yes, I was thinking on this as well. 
One feedback item is for the UI the angle is + or - for forward or reverse, so the UI would present total steps -360 to 360 or step size -180 to 180. Does this sounds reasonable or should angle still be presented in the UI and steps deduced from this ?